core/src/main/java/io/grpc/util/OutlierDetectionLoadBalancer.java

```diff
@@ -733,6 +765,9 @@ public void ejectOutliers(AddressTrackerMap trackerMap, long ejectionTimeNanos)
       double requiredSuccessRate =
           mean - stdev * (config.successRateEjection.stdevFactor / 1000f);
 
+      logger.log(ChannelLogLevel.DEBUG,
```

Comment:
Do we really need this log statement? By default it would trigger every ten seconds and would mostly just be spam.

Reply:
I added this to help users debug cases when they don't understand why SuccessRate algorithm ejected/unejected some hosts. 
What do you think if I add `mean: {0}, stdev: {1}, requiredSuccessRate: {2}' to the log message on line 782 instead? This will be duplicated for all ejected hosts, but I don't think this is a big deal.

Reply:
How about incorporating it in the acceptResolvedAddresses() method? That is where we receive any new configs, you could just log once whenever these values change.

Reply:
I could be missing something, but my reading of the code is that successRates is dynamically calculated at runtime [here](https://github.com/grpc/grpc-java/blob/8ab4fa0a8436e8509cdf49a270d32958c6b7266f/core/src/main/java/io/grpc/util/OutlierDetectionLoadBalancer.java#L758-L761) based on the individual stream success rates.  So we don't know this information in the acceptResolvedAddresses method and it could change between ejectOutliers method invocations.

Reply:
So the use-case I am thinking about is that mean success rate can drop for all upstreams, which could cause previously ejected outliers to be unelected. This particular log message is meant to be helpful in such scenarios.

Reply:
Ah of course, you are right. For a moment I was thinking we are logging static things from the config. 

Let's go with your suggestion and log this info when we eject, instead of here.

Reply:
done.